sdk/modelsrepository/Azure.Iot.ModelsRepository/samples/readme.md

```diff
@@ -0,0 +1,148 @@
+# Introduction
+
+The Azure IoT Models Repository enables builders to manage and share digital twin models for global consumption. The models are [JSON-LD][json_ld_reference] documents defined using the Digital Twins Definition Language ([DTDL][dtdlv2_reference]).
+
+For more info about the Azure IoT Models Repository checkout the [docs][modelsrepository_msdocs].
+
+# IoT Models Repository Samples
```

Comment:
Markdown files should only have 1 level-1 header.

Reply:
I'll fix -- I drew inspiration from the DigitalTwins samples readme which currently has the same double level-1 header https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md#introduction .

Reply:
Oops, @azabbasi perhaps you can fix that when you get a chance. Markdown files are supposed to have only 1 top level header.